src/cow-react/common/services/getQuoteCurrency/index.ts

```diff
@@ -0,0 +1,50 @@
+import { SupportedChainId } from 'constants/chains'
+import { Currency, CurrencyAmount } from '@uniswap/sdk-core'
+import { DAI, USDC_MAINNET, USDT } from 'constants/tokens'
+import { DAI_RINKEBY, USDT_RINKEBY, USDC_RINKEBY } from 'utils/rinkeby/constants'
+import { DAI_GOERLI, USDT_GOERLI, USDC_GOERLI } from 'utils/goerli/constants'
+import { USDC_GNOSIS_CHAIN, USDT_GNOSIS_CHAIN, WXDAI } from 'utils/gnosis_chain/constants'
+import { NATIVE_CURRENCY_BUY_ADDRESS } from 'constants/index'
+
+// TODO: Find a solution for using API: https://www.coingecko.com/en/categories/stablecoins
+const STABLE_COINS: { [key in SupportedChainId]: string[] } = {
```

Comment:
Maybe we should move this to `custom/constants/index.ts`?

Reply:
It's a temporary solution, for sure we will change it. Given this, I think it doesn't make sense.